addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -597,3 +597,13 @@ def test_bank_details_import(self):
         self.env['account.edi.common']._import_retrieve_and_fill_partner_bank_details(invoice, [acc_number])
         self.assertEqual(invoice.partner_bank_id, partner_bank, "Partner bank must be the same")
         self.assertTrue(partner_bank.active, "Partner bank must be the activated")
+
+    def test_bank_details_import_duplicate(self):
+        acc_number = '1234567890'
+        invoice = self.env['account.move'].create({
+            'partner_id': self.partner_a.id,
+            'move_type': 'in_invoice',
+            'invoice_line_ids': [Command.create({'product_id': self.product_a.id})],
+        })
+        # We will not try to create multiple partner bank records with the same account number
```

Comment:
Just a small suggestion because I found it a bit hard to understand at first, but up to you!


```suggestion
        # We should not create multiple partner bank records with the same account number
```

Reply:
I clarified the comment a bit (and rebased)